kmir/k-src/mir.md

```diff
@@ -71,6 +70,11 @@ module MIR-SYMBOLIC
   imports MIR-EXECUTION
   imports MIR-FINALIZATION
 
+    syntax KItem ::= runLemma ( Step ) | doneLemma ( Step )
+    // -------------------------------------------------------
+    rule <k> runLemma(S) => doneLemma(S) ... </k>
+
+    syntax Step ::= Bool | Int | MIRValue
```

Comment:
I imagine it is not ideal to have this in `MIR-SYMBOLIC`, in truth I was running into errors trying to add this in any other file because of adding declarations in proof module. Exactly how to avoid that error I am unsure of.